sharktank/tests/ops/sharded_test.py

```diff
@@ -25,7 +25,7 @@ def testAllGather(self):
         ]
         expected_result = torch.cat(shards, dim=shard_dim)
 
-        sharded = SplitPrimitiveTensor(shard_dim=shard_dim, ts=shards)
+        sharded = SplitPrimitiveTensor(shard_dim=shard_dim, ts=shards, devices=tuple(range(shard_count)), devices_pinned=False)
```

Comment:
Here and below. Make all the existing tests work by default rather than changing their behavior. We want to make PP invisible when possible